pkgs/development/tools/misc/tideways/daemon.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenvNoCC
+, lib
+, fetchurl
+}:
+
+stdenvNoCC.mkDerivation rec {
```

Comment:
Do you think it would be possible implement the [finalAttrs pattern](https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes)  and remove the `rec` keyword ?